api/admin.py

```diff
@@ -65,5 +66,7 @@ async def main(args):
                         help="Admin group name")
     parser.add_argument('--database', default='kernelci',
                         help="KernelCI database name")
+    parser.add_argument('email',
+                        help="Admin user email address")
```

Comment:
All the other args are non-positional so for consistency:
```suggestion
    parser.add_argument('--email', required=True,
                        help="Admin user email address")
```